internal/datastore/spanner/migrations/zz_migration.0010_add_transaction_metadata_table.go

```diff
@@ -0,0 +1,34 @@
+package migrations
+
+import (
+	"context"
+
+	"cloud.google.com/go/spanner/admin/database/apiv1/databasepb"
+)
+
+const (
+	addTransactionMetadataTable = `CREATE TABLE transaction_metadata (
+			transaction_tag STRING(36) NOT NULL,
+			created_at TIMESTAMP DEFAULT (CURRENT_TIMESTAMP()),
+			metadata JSON
+		) PRIMARY KEY (transaction_tag),
+		ROW DELETION POLICY (OLDER_THAN(created_at, INTERVAL 2 DAY))
```

Comment:
Right now revisions will be around for an hour, which is the [default value](https://cloud.google.com/spanner/docs/timestamp-bounds#maximum_timestamp_staleness).

However, the way clients will retrieve the data is via the Watch API, and the table deltas are materialized into a change stream that has longer retention, [one day by default](https://github.com/authzed/spicedb/issues/1457). 

So I'm confused as to why we let the metadata sit around for 2 days, they will only be able to retrieve it for up to 1 day. Something like 25 hours should be sufficient.

Reply:
Only days is supported, so it had to be 1 day or 2:

> num_days is the number of days past the timestamp in the timestamp_column in which the row is marked for deletion. The value must be a non-negative integer and DAY is the only supported unit.

Reply:
Please add a comment to clarify this then.